java.lang.NoSuchMethodError: No virtual method getContentLengthLong()

Seeing this on Android 5.1. Apparently `HttpURLConnection.getContentLengthLong` doesn't exist on old Android versions.

```
Caused by: java.lang.NoSuchMethodError: No virtual method getContentLengthLong()J in class Ljava/net/HttpURLConnection; or its super classes (declaration of 'java.net.HttpURLConnection' appears in /system/framework/core-libart.jar)
	at org.dmfs.httpessentials.httpurlconnection.HttpUrlConnectionResponseEntity.contentLength(HttpUrlConnectionResponseEntity.java:60)
	at org.dmfs.httpessentials.responsehandlers.StringResponseHandler.handleResponse(StringResponseHandler.java:86)
```

--- a/httpurlconnection-executor/src/main/java/org/dmfs/httpessentials/httpurlconnection/HttpUrlConnectionResponseEntity.java
+++ b/httpurlconnection-executor/src/main/java/org/dmfs/httpessentials/httpurlconnection/HttpUrlConnectionResponseEntity.java
@@ -57,7 +57,16 @@ final class HttpUrlConnectionResponseEntity implements HttpResponseEntity
     @Override
     public Optional<Long> contentLength()
     {
-        long length = mConnection.getContentLengthLong();
+        long length;
+        try
+        {
+            length = mConnection.getContentLengthLong();
+        }
+        catch (NoSuchMethodError e)
+        {
+            // getContentLengthLong has been added in Java 7 and Android SDK 24, fall back to integer on older runtime engines
+            length = mConnection.getContentLength();
+        }
         return length < 0 ? Absent.<Long>absent() : new Present<>(length);
     }
 

